openlr_dereferencer/decoding/__init__.py

```diff
@@ -25,7 +24,21 @@ def decode(reference: LineLocationRef, reader: MapReader, radius: float = SEARCH
         radius: The search path for the location's components' candidates
 
     Returns:
-        The returned value will be a dereferenced openlr_dereferencer.LineLocation. It contains
-        a list of Line objects, which is trimmed by the offset values of the line location.
+        This function will return one or more map object, optionally wrapped into some class.
+        Here is an overview for what reference type will result in which return type:
+
+        reference                     | returns
+        ------------------------------|-------------------------------
+        openlr.GeoCoordinateLocation  | Node
```

Comment:
maybe we can get the type names changed to `LineLocationReference` and `PointAlongLineLocationReference`

Reply:
- [ ] Let `python-openlr` names conform to whitepaper terminology